### PR TITLE
fix: remove all @ts-ignore comments in the mentioned components 

### PR DESCRIPTION
**Pull request by nlok5923**
Refers: #2798 

**What kind of change does this PR introduce?**

remove all @ts-ignore comments in the mentioned components .

 - Accessory

**Did you add tests for your changes?**

**If relevant, did you update the documentation?**

**Summary**
Removed all @ts-ignore comments

**Does this PR introduce a breaking change?**
No

**Other information**

### CODE REVIEW

**Comment by nlok5923** [comment]
@flyingcircle sir please this pull request sir while npm test it works fine and passed all test

**Comment by RohitKumar-200** [comment]
You need to fix the code. Just removing //@ts-ignore will not work.

**Comment by nlok5923** [comment]
@RohitKumar-200 please can you tell me how can i compile the code to know the error which @ts-ignore hides

**Comment by RohitKumar-200** [comment]
Your code editor will automatically show errors when you remove //@ts-ignore. 
Example:
error
If you are using vs-code, you can see all details about it in the problem section, or just hovering on that word will give details about that.
I hope it helps.

**Comment by nlok5923** [comment]
Got it @RohitKumar-200 thanks

File: src/config/ThemeProvider.tsx [hunk 1]
```diff
@@ -12,7 +12,6 @@ export interface ThemeProps<T> {
   replaceTheme: (updates: RecursivePartial<FullTheme>) => void;
 }
 
-// @ts-ignore
 export const ThemeContext: React.Context<ThemeProps<{}>> = React.createContext({
```
Comment (nlok5923) [inline src/config/ThemeProvider.tsx:15]: @pranshuchittora sir here i tried to remove ThemeProps interface which result in test faliure as this components is utilized in withTheme components can you please provide me clue to resolve error .

**Comment by murat-mehmet** [comment]
Well this definitely needs a fix, most components are broken in TS right now, if not all. I have to wrap the components with correct props to bypass the problem
```
import React, {Component} from 'react';
import {Icon as IconOriginal} from "react-native-elements";
import {IconProps} from "react-native-elements/dist/icons/Icon";

export class Icon extends Component<IconProps> {
    render() {
        return (
            <IconOriginal {...this.props as any} />
        );
    }
}
```

Any other workaround?

**Comment by nlok5923** [comment]
@flyingcircle sir the //@ts-ignore is removed from accessory component also in accessory components removal of //@ts-ignore returns no error.

please review the changes

**Comment by snitin315** [comment]
Please change `Fixes` to `Refers`, as we don't want to close the issue until it's all done.

**Comment by flyingcircle** [comment]
Accessory tag was removed in another PR, thanks for the work! I hope you continue to fix issues in other files.

**Pull request closed without merging**